engine/src/cmd/collide2/opcodesysdef.h

```diff
@@ -1,7 +1,9 @@
 /**
  * opcodesysdef.h
  *
+ * Copyright (C) Daniel Horn
  * Copyright (C) 2020 pyramid3d, Stephen G. Tuggy, and other Vega Strike contributors.
+ * Copyright (C) 2021 Stephen G. Tuggy
```

Comment:
It does not look as if you've made any changes to this file.
I don't care that you added the line but I wonder if you meant to make more changes and something got lost somewhere.

Reply:
No, I don't think there are any changes missing. Thanks for checking